index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <section class="middle">
+        <header class="navbar">
+            <ul class="menu">
+                <li><a href="">Portfolio</a></li>
+                <li><a href="">About</a></li>
+                <li><a href="">Contact</a></li>
+            </ul>
+        </header>
+        <div>
+            <h1>Hey there. I'm Maximiliano.</h1>
+            <h2 class="secondtitle">I'm a software developer</h2>
+        </div>
+        <p><strong>I can help you build a product , feature or website Look through some of my work and<br>experience! If you like what you see and have a project you need coded,<br>don’t hestiate to contact me.</strong></p>
+        <ul class="row">
+            <li class="column">
+                <a href=""><img src="images/github.png" alt="github"></a>
+            </li>
+            <li class="column">
+                <a href=""><img src="images/Linkedin.png" alt="linkedin"></a>
+            </li>
+            <li class="column">
+                <a href=""><img src="images/angellist.png" alt="angellist"></a>
+            </li>
+            <li class="column">
+                <a href=""><img src="images/twitter.png" alt="twitter"></a>
+            </li>
+            <li class="column">
+                <a href=""><img src="images/medium.png" alt="medium"></a>
+            </li>
+        </ul>
+    </section>
+    <section>
+        <div class="spacing">
+            <a class="disabled" href=""><img src="images/Disabled.png" alt="next"></a>
+        </div>
+        <h2 class="subtitle2">My Recent Works</h2>
```

Comment:
- [ ] Kindly fix the work section
- Your Design
![Capturedest](https://user-images.githubusercontent.com/81444338/171761350-d311e9e0-53a4-4a15-87fa-b155051a1945.PNG)
- Figma Design
![Capturefdes](https://user-images.githubusercontent.com/81444338/171761653-c5c7a2c8-dbe5-4603-be91-19519dca88ed.PNG)